.github/workflows/renovate.yml

```diff
@@ -37,9 +37,9 @@ jobs:
           # renovate will generate an onboarding PR after the first run
           RENOVATE_ONBOARDING: "true"
           # username needed for Github auth (Github app name + "[bot]")
-          RENOVATE_USERNAME: "kubearchive-renovate[bot]"
+          RENOVATE_USERNAME: "kronicler-renovate[bot]"
```

Comment:
@skoved for this we would need to recreate the bot, right? If that is the case, comment something on the issue linked to this PR, so we have there "post-tasks" we need to do.

Reply:
> we have there "post-tasks" we need to do.

I would say these are "pre-task" things. They can be done ahead of time, and things will just work after this is committed.  I'll start a list in the description for one other thing, and @skoved can it it there.

Reply:
> I'll start a list in the description for one other thing

I see @rh-hemartin already did this, and he even added the one I was thinking of.  I already had to create a `kronicler` quay.io org and put in the `flog` image for the tests to run.

Reply:
[I added a comment about it to the issue for this PR](https://github.com/kubearchive/kubearchive/issues/966#issuecomment-2761328034)